src/subscription_manager/migrate/migrate.py

```diff
@@ -243,7 +246,7 @@ def check_ok_to_proceed(self, username):
             system_exit(1, msgs)
 
         try:
-            self.cp.getOwnerList(username)
+            self.cp.getStatus()
```

Comment:
is getStatus() just being used as a consumer auth check here?

Or it's side effects needed (creating status cache)